PTBR/Recorded_Media_PTBR.txt

```diff
@@ -2217,130 +2217,130 @@ RM_b9bfda4d-de97-424a-b3ba-84a0ea74ff3b = "Goodbye... asshole."
-RM_daa60b5b-43c9-4cfb-8f23-dc48853d5043 = "Throw in your bacon, your tomato, your onion..."
-RM_ed376572-7701-4ca5-a925-67cea8864a03 = "... your carrots, if you want."
-RM_d37a66f0-9d10-4461-b57e-1e685be3b5d2 = "Then, if you're like me, just a bit of salt for flavor."
-RM_585cf31c-779d-4adb-8454-c2582e311400 = "Seventy minutes on high heat will do ya."
-RM_d942418f-501e-4136-9ffd-84154e0f8b10 = "Bam! Looks good, huh?"
-RM_8b2917e8-6d8a-44c1-b730-d13027098890 = "Split into four bowls, or two if you're hungry."
-RM_b9d75870-8037-4f0a-8b26-8343a25c4ec3 = "See y'all tomorrow."
-RM_89bb626a-ac81-4dd7-bb60-aa8838601ce5 = "VHS: The Cook Show E4"
-RM_9862d8ea-2f5d-49ab-a0ea-8840255588fc = "Episode 4: Salads"
-RM_e6b4d102-e3d2-48ee-9119-2af1b69c296c = "Now, I stood on the scales the other day. ."
-RM_a5d604b2-9c6e-404e-acb2-35e70955a2e2 = "I got a shock people!"
-RM_21b985b6-458c-4de4-9236-05590b3c14b3 = "Got me to thinking about a show that celebrates the healthy life?"
-RM_62e24c2f-4a16-4ce1-968c-9d761f9c2559 = "So here it is - the Salad Special."
-RM_66994448-f2d8-4b1f-9070-a4cbadd90e66 = "Just cause there's leaves, don't mean you'll heave."
-RM_a97f2f84-e464-4968-93bb-c451fd76ba07 = "Take a bowl - and as a base throw in some lettuce."
-RM_74678a79-c45a-48c0-a668-81f779b7f366 = "Then we add cheese, ham - and we're almost at Bam!"
-RM_21464593-7f45-491a-9fdf-d1c309ed5113 = "Let's add in some mayo..."
-RM_bd2a5cc3-438f-4936-9f86-3bfcd447f40a = "Maybe a bit more mayo..."
+RM_a5341933-860e-4020-9f37-97f77432a699 = "Episódio 1: Bolo"
+RM_64318b86-aece-4f80-b401-ae62f5190e39 = "Como você está? Bem-vindo de volta ao The Cook Show!"
+RM_a0eede9d-e5fd-493b-a191-f073e4121ac5 = "Trazido a você por Kitten Knives - Elegância em utensílios de cozinha! De origem americana."
+RM_e30245f3-70ad-4503-b636-ecb303b8c08e = "Sem mais delongas!"
+RM_92b85178-ee18-4a5e-a67a-a184c61d0a75 = "Você sabe porque estamos aqui.."
+RM_1d72cef4-a4ac-4dcd-9679-ed4650d443c3 = "Bolo!"
+RM_7d3ce2ec-634e-427f-b5aa-91955a16bed7 = "Primeiro você precisa de uma massa - pegue uma tigela assim e misture..."
+RM_8784d640-f175-4ee0-97fb-f74d492842aa = "Farinha de trigo! Manteiga! Açúcar! Ovo! Fermento! Leite!"
+RM_03ea6a81-c7c9-4302-82eb-33d5725f7f83 = "Consegue isso tudo?"
+RM_f544b33e-5483-43e4-97fc-a6fe5b02d67d = "Adicione tudo na assadeira e... adicione um pouco de sabor."
+RM_0a6df861-e39c-4bb9-888f-ff764a5d2ab9 = "Hoje estou usando banana, frutas vermelhas..."
+RM_88108679-c51d-444f-8de8-9af84a3b4c4e = "E uma pitada de limão."
+RM_9bc8656b-b3c3-4116-b6d7-2037d823751b = "Depois leve ao forno, e vai ficar uma delícia!"
+RM_387f7039-f6f2-4f15-8ee0-0348efab3bf9 = "Impossível vencer esse cheiro, hein?"
+RM_0dc6af6b-f478-458c-ab10-7a541663cbde = "Te vejo amanhã!"
+RM_08265ea3-f212-46ca-a1c8-467d50460ce6 = "VHS: The Cook Show EP2"
+RM_9b10a612-7d37-4845-a35f-e61130fbe8da = "Episódio 2: Sopa"
+RM_97c2b005-f0d1-4227-8cc5-06e6fa5aee4a = "Bem-vindos de volta ao The Cook Show, pessoal."
```

Comment:
**Bem-vindo** de volta ao The Cook Show, pessoal.

- Não flexiona para o plural. Basta imaginar como "Bem-vindo, pessoal!".